distribution/ddf-common/src/main/resources/common-bin.xml

```diff
@@ -56,6 +56,11 @@
                 <exclude>karaf-manual*.html</exclude>
                 <exclude>karaf-manual*.pdf</exclude>
 
+                <!--
+                Skipping customized libraries.
+                -->
+                <exclude>**/org/apache/aries/blueprint/org.apache.aries.blueprint.core/1.8.2/org.apache.aries.blueprint.core-1.8.2.jar</exclude>
```

Comment:
✏️ can you wildcard the version numbers?

Reply:
I could, but I'm hoping this is a relatively short-lived workaround.

Reply:
The maven resources plugin should get run on this file so we should be able to swap out the versions for a variable.